### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by noodlemctwoodle**
Updated ReadMe.md to use system environment variables for allowing direct browsing to plugins directory.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ It achieves this in 2 ways:
 
 ### Installation
 
-> **TL;DR** Copy contents of `dist/` to `%AppData%/Local/Finkit/ManicTime/Plugins/Packages`.
+> **TL;DR** Copy contents of `dist/` to `%systemdrive%\Users\%username%\AppData\Local\Finkit\ManicTime\Plugins`.
```
Comment (mikefitz888) [inline readme.md]: Could these be '/' instead? So that it is consistent with the rest of the document.

Comment (noodlemctwoodle) [reply readme.md]: No, dont be stupid! :D

Comment (mikefitz888) [reply readme.md]: Could you change all the '/' to '\\' please? To keep the document consistent?

Comment (noodlemctwoodle) [reply readme.md]: I would love too.

Comment (noodlemctwoodle) [reply readme.md]: Added updated changes

File: readme.md [hunk 2]
```diff
@@ -15,16 +15,16 @@ It achieves this in 2 ways:
 
 ### Installation
 
-> **TL;DR** Copy contents of `dist/` to `%systemdrive%\Users\%username%\AppData\Local\Finkit\ManicTime\Plugins`.
+> **TL;DR** Copy contents of `\dist` to  `%systemdrive%\Users\%username%\AppData\Local\Finkit\ManicTime\Plugins`.
```
Comment (mikefitz888) [inline readme.md]: extra space after 'to'

File: readme.md [hunk 3]
```diff
@@ -15,16 +15,16 @@ It achieves this in 2 ways:
...
 
 First step is to locate the ManicTime plugin directory. To do so, open ManicTime. Click on the cog at the top right of the window, and click
-`Advanced > Open Datebase Folder` in the menu that appears. You will see a folder here called `Plugins`, navigate to `Plugins/Packages`. Here is where you will copy the plugin folder.
+`Advanced > Open Datebase Folder` in the menu that appears. You will see a folder here called `Plugins`, navigate to `Plugins\Packages`. Here is where you will copy the plugin folder. If Packages does not exist, create it.
```
Comment (mikefitz888) [inline readme.md]: Format Packages within backticks.

**Comment by noodlemctwoodle** [comment]
Corrected \Packages and removed additional space.

**Comment by noodlemctwoodle** [comment]
Corrected `\Packages`

**Pull request merged**